tests/unit/_internal/runner/test_strategy.py

```diff
@@ -0,0 +1,31 @@
+import os
+import typing as t
+
+import bentoml
+from bentoml._internal.runner import strategy
+from bentoml._internal.resource import get_resource as resource_get_resource
+from bentoml._internal.runner.strategy import DefaultStrategy
+
+
+class GPURunnable(bentoml.Runnable):
+    SUPPORTED_RESOURCES = ("nvidia.com/gpu",)
+
+
+def unvalidated_get_resource(x: t.Dict[str, t.Any], y: str):
+    return resource_get_resource(x, y, validate=False)
+
+
+def test_default_gpu_strategy(monkeypatch):
```

Comment:
Types of monkeypatch is
```
if TYPE_CHECKING:
    from _pytest.monkeypatch import MonkeyPatch
```

Reply:
done!
1 Q: the if TYPE_CHECKING is a check to improve perf type checker right?